c2corg_api/views/user.py

```diff
@@ -372,6 +372,7 @@ def get(self):
             'name': user.name,
             'forum_username': user.forum_username,
             'id': user.id
+
```

Comment:
@tsauerwein, user id should be get on log in, not when editing the user account.